docs/sdk/src/reference_docs/mod.rs

```diff
@@ -32,6 +32,9 @@ pub mod glossary;
 /// Learn about the WASM meta-protocol of all Substrate-based chains.
 pub mod wasm_meta_protocol;
 
+/// Learn about the token-related logic in Substrate and how to apply it to your use case.
+pub mod tokens_in_substrate;
```

Comment:
```suggestion
pub mod frame_tokens;
```

as in, this is all FRAME concepts, not substrate, right? 

and for now all frame ref docs are prefixed with `frame_`.

We should someday move them to `mod frame`.

Reply:
In fact there is already `pub mod frame_currency;`.